google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -514,6 +514,15 @@ def file path, generation: nil, encryption_key: nil
         # @param [Hash] metadata A hash of custom, user-provided web-safe keys
         #   and arbitrary string values that will returned with requests for the
         #   file as "x-goog-meta-" response headers.
+        # @param [Symbol, String] storage_class Storage class of the file.
+        #   Determines how the file is stored and determines the SLA and the
+        #   cost of storage. Values include `:multi_regional`, `:regional`,
+        #   `:nearline`, `:coldline`, `:standard`, and `:dra` (Durable Reduced
```

Comment:
AFAIK, `STANDARD` is the same as `REGIONAL` or `MULTI_REGIONAL`, depending on the bucket's location. Meaning, the behavior is different depending if it is in the US or Asia.